recipes/common/Makefile.common

```diff
@@ -15,6 +15,10 @@ ARCH ?=
 CONTAINERFILE ?= Containerfile
 GRAPH_ROOT=$(shell podman info --format '{{ .Store.GraphRoot }}')
 
+DISK_USER ?=
```

Comment:
Why not default these to
 DISK_USER=$(id -u) DISK_GROUP=$(id -g)

Reply:
Hello @rhatdan, with sudo, the makefile is executed by the root user, so the default value in the Makefile is meaningless. We want to change the disk image to non-root owner, if set default value, bib will add `--chown 0:0`.

Here is an example:
```bash
$ cat Makefile
DISK_USER ?= $(shell id -u)
DISK_GROUP ?= $(shell id -g)

.PHONY: show-ids

show-ids:
	@echo "UID: $(DISK_USER)"
	@echo "GID: $(DISK_GROUP)"

$ make
UID: 1001
GID: 1001

$ sudo make
UID: 0
GID: 0

$ sudo make DISK_USER=$(id -u) DISK_GROUP=$(id -g)
UID: 1001
GID: 1001
```

Reply:
Right but for non root users, they don't have to think about it and the right thing happens.
For rootful users this becomes a noop chown root to root.